aikau/src/test/resources/testApp/WEB-INF/classes/alfresco/site-webscripts/alfresco/forms/controls/BaseForm.get.js

```diff
@@ -172,6 +172,34 @@ model.jsonModel = {
                         }
                      ]
                   }
+               },
+               {
+                  name: "alfresco/forms/Form",
+                  id: "FOCUS_AND_SUBMIT_ON_ENTER_FORM",
+                  config: {
+                     okButtonPublishTopic: "SUBMIT_FOCUS_AND_SUBMIT_ON_ENTER_FORM",
+                     okButtonPublishGlobal: true,
+                     firstFieldFocusOnLoad: true,
+                     submitOnEnter: true,
+                     widgets: [
+                        {
+                           id: "FORM_FIELD1",
+                           name: "alfresco/forms/controls/TextBox",
```

Comment:
We should be testing a range of widgets here - not just text box, in particular we need consideration over what happens with widgets that have their own keypress listeners.